tbx/static_src/sass/components/_clip-letter.scss

```diff
@@ -0,0 +1,23 @@
+@use "config" as *;
+
+.clip-letter {
+    -webkit-text-fill-color: transparent;
+    background: var(--letter-motif), no-repeat;
+    background-size: 200% 150%;
```

Comment:
It's hard to avoid being arbitrary here as different background sizes look good on different letters. I noticed on the I the background-image was very small because 200% width on a I is still quite narrow. I just googled to see if you can set a min-value for background size and the last answer on https://stackoverflow.com/questions/10682765/does-css3-offer-a-minimum-size-property-for-background looks promising. Noting that this will need to be different when this is used with different font-sizes though.